x-pack/plugins/files/server/usage/counters.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export function getCounters(prefix: string) {
+  return {
+    DELETE: `${prefix}_delete`,
+    DELETE_ERROR: `${prefix}_delete_error`,
```

Comment:
Would it be possible to get more context to the error counter names, such as an error code? I think we could get more value by using event names for errors in the form of: `eventType:error:errorCode`.

Example: `csv_download:error:403`

Reply:
That's a really good idea @tsullivan!

I've mapped out some known errors that I think would be of interest. The download case is a little bit hard because the error space is not very well mapped out. Let me know what you think!